### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ABahatiy**
https://abahatiy.github.io/layout_product-cards/ 
https://abahatiy.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/stars.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+.card__stars {
```
Comment (anstsot) [inline src/stars.css:2]: it should be separate BEM block `stars`
```suggestion
.stars {
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  --card-width: 200px;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.card {
```
Comment (anstsot) [inline src/style.css:20]: create separate `card` css file
image

**Review by anstsot — CHANGES_REQUESTED** [review]
It’s almost done! Let’s improve a little

**Comment by ABahatiy** [comment]
Thank you for review. Please check a new changes

File: src/card.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.card {
+  box-sizing: border-box;
```
Comment (IvanFesenko) [inline src/card.css:8]: move box-sizing into `*` selector

File: src/card.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--main-accent);
+}
+
+.card__code {
+  padding-bottom: 16px;
+  font-weight: 400;
```
Comment (IvanFesenko) [inline src/card.css:36]: font-weight by default is 400 - redudant. 
Fix all similar cases

File: src/card.css [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-accent);
+}
+
+.card__price-value {
+  padding-bottom: 16px;
+  font-weight: 700;
+  font-size: 16px;
```
Comment (IvanFesenko) [inline src/card.css:62]: font-size by default is 16px - redudant too as well

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,73 @@
...
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="card__picture"
+      >
+
+      <div class="card__model">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__code">Product code: 195434</div>
```
Comment (IvanFesenko) [inline src/index.html:47]: Use semantic tags for text content
h2-h5 for heading, p for other text

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Check my comments

**Review by etojeDenys — APPROVED** [review]
lgtm

**Pull request closed without merging**